src/web_app/src/pages/log_page.tsx

```diff
@@ -0,0 +1,91 @@
+/* eslint-disable @typescript-eslint/no-floating-promises */
+import Container from "@mui/material/Container";
+import CssBaseline from "@mui/material/CssBaseline";
+import GlobalStyles from "@mui/material/GlobalStyles";
+import Typography from "@mui/material/Typography";
+import Table from "@mui/material/Table";
+import TableBody from "@mui/material/TableBody";
+import TableCell from "@mui/material/TableCell";
+import TableContainer from "@mui/material/TableContainer";
+import TableHead from "@mui/material/TableHead";
+import TableRow from "@mui/material/TableRow";
+import React, {useEffect} from "react";
+import {AppbarStyle} from "../components/appBar";
+import http from "../http-common";
+import {Types} from "../types/data";
+import Paper from "@mui/material/Paper";
+import Row from "../components/log_row";
+
+export function LogsPage() {
+    const [
+        logs,
+        setLogs
+    ] = React.useState<Array<Types.Log>>([
+        {
+            "id"    : "",
+            "method": "",
+            "logs"  : []
+        }
+    ]);
+
+    useEffect(() => {
+        (
+            async() => {
+                await http.get("get_logs")
+                    .then(response => {
+                        // eslint-disable-next-line @typescript-eslint/no-unsafe-argument, @typescript-eslint/no-unsafe-member-access
+                        console.log(response.data.data);
+                        // eslint-disable-next-line @typescript-eslint/no-unsafe-argument, @typescript-eslint/no-unsafe-member-access
+                        setLogs(response.data.data);
+                    });
+            }
+        )();
```

Comment:
haha so I will change the name of my key(the second data) in the backend. Or just removing it according to what the new version of the flask api send us.

Reply:
I think I should have explained way more. I am sorry, just saying "that's illegal" is far from enough. 

The part that made me bug wasn't the name of the key, it was the asynchronous function inside the [IIFE](https://developer.mozilla.org/fr/docs/Glossary/IIFE) with in it an asynchronous function. Is the asynchronous function returned or directly executed ? What's the point of the IIFE here, since it is the only statement inside a function body ? 
If the asynchronous function is executed, then you could write it directly in the `useEffect` call as `useEffect` support asynchronous callback. 
If it is returned, then add a return statement and remove the IIFE as it serve no purpose as far as I understand your code.